src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -11793,7 +11756,6 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
         // Evaporator (IU side) operational parameters
         Pevap = this->refrig->getSatPressure(state, this->IUEvaporatingTemp, RoutineName);
         Psuction = Pevap;
-        Tsuction = this->IUEvaporatingTemp;
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:11917]:(style),[redundantAssignment],Variable 'Tsuction' is reassigned a value before the old one has been used.